diff -r -U2 bundle-sources.org/org/eclipse/osgi/internal/framework/legacy/PackageAdminImpl.java bundle-sources/org/eclipse/osgi/internal/framework/legacy/PackageAdminImpl.java
--- bundle-sources.org/org/eclipse/osgi/internal/framework/legacy/PackageAdminImpl.java	2018-10-18 22:00:59.000000000 +0200
+++ bundle-sources/org/eclipse/osgi/internal/framework/legacy/PackageAdminImpl.java	2018-10-18 22:01:08.000000000 +0200
@@ -296,5 +296,5 @@
 	}
 
-	public Bundle getBundle(final Class<?> clazz) {
+	public Bundle getBundle(final Class clazz) {
 		if (System.getSecurityManager() == null)
 			return getBundlePriv(clazz);
